packages/legend-query-builder/src/stores/explorer/QueryBuilderPropertySearchState.ts

```diff
@@ -224,6 +289,64 @@ export class QueryBuilderPropertySearchState {
       }
       return true;
     });
+    return filteredTest;
+  }
+
+  getFilteredPropertyNodes(): Promise<QueryBuilderExplorerTreeNodeData[]> {
+    return new Promise((resolve) => {
```

Comment:
why do we make this a `promise`? Also, why not make this a `getter` as discussed?

Reply:
whoops I messed up rebase process this was kept the get is there elsewhere 😓 replaced